kornia/geometry/transform/imgwarp.py

```diff
@@ -268,6 +269,37 @@ def ay(p, q):
     return M.view(-1, 3, 3)  # Bx3x3
 
 
+def angle_to_rotation_matrix(angle: torch.Tensor,
+                             do_deg2rad: bool = False) -> torch.Tensor:
+    """
+    Creates a rotation matrix out of angles
+    Args:
+        angle: (torch.Tensor): tensor of angles, any shape.
+        do_deg2rad: (bool): if we should convert to radians first
+
+    Returns:
+        torch.Tensor: tensor of Nx2x2 rotation matrices.
+
+    Shape:
+        - Input: :math:`(*)`
+        - Output: :math:`(*, 2, 2)`
+
+    Example:
+        >>> input = torch.rand(1, 3)  # Nx3
+        >>> output = kornia.angle_to_rotation_matrix(input)  # Nx3x2x2
+    """
+    if do_deg2rad:
+        ang = kornia.deg2rad(angle)
+    else:
+        ang = angle
+    n_dims = len(ang.size())
+    cos_a = torch.cos(ang).unsqueeze(-1).unsqueeze(-1)
+    sin_a = torch.sin(ang).unsqueeze(-1).unsqueeze(-1)
+    A1_ang = torch.cat([cos_a, sin_a], dim=n_dims + 1)
+    A2_ang = torch.cat([-sin_a, cos_a], dim=n_dims + 1)
+    return torch.cat([A1_ang, A2_ang], dim=n_dims)
```

Comment:
1) I'd avoid the `do_deg2rad`, if we specify the format of the input data in the documentation is enough and will make the signature much cleaner. Similar to `deg2rad` or other convention functions.

2) to simply a bit and avoid intermediate variables. JIT might raise errors since `n_dims` will be casted to tensor :(
I propose this:
```python
def angle_to_rotation_matrix(angle: torch.Tensor) -> torch.Tensor
    """Creates a rotation matrix out of angle in radians.

     Add the existing docs.
     """
    cos_a: torch.Tensor = torch.cos(ang)
    sin_a: torch.Tensor = torch.sin(ang)
    return torch.stack([cos_a, sin_a, -sin_a, cos_a], dim=-1).view(*x.shape, 2, 2)
```